### PR TITLE
adding retries in get_data to log statements, expanding exception scope

### PR DESCRIPTION
**Pull request by willkjackson**
This PR adds ReadTimeoutError and ConnectionError to the FPDS retries. Currently we were only checking for ConnectionResetError. The error I've been seeing during the nightly job for this is requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), which would not be caught (or retried) and therefore immediately fail in the current version of the script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # DATA Act Broker Backend
 
-The DATA Act broker backend is a collection of services that power the DATA Act's central data submission platform.
+The DATA Act broker backend is a collection of services that power the DATA Act's central data submission platform.
```
Comment (alburde1) [inline README.md]: Why was this added?

File: dataactcore/scripts/pullFPDSData.py [hunk 2]
```diff
@@ -1340,12 +1340,26 @@ def get_delete_data(contract_type, now, sess, last_run, start_date=None, end_dat
     i = 0
     logger.info('Starting delete feed: %sCONTRACT_TYPE:"%s"', delete_url + params, contract_type.upper())
     while True:
-        resp = requests.get(delete_url + params + 'CONTRACT_TYPE:"' + contract_type.upper() + '"&start=' + str(i),
-                            timeout=60)
-        resp_data = xmltodict.parse(resp.text, process_namespaces=True,
-                                    namespaces={'http://www.fpdsng.com/FPDS': None,
-                                                'http://www.w3.org/2005/Atom': None,
-                                                'https://www.fpds.gov/FPDS': None})
+        exception_retries = -1
```
Comment (alburde1) [inline dataactcore/scripts/pullFPDSData.py]: Are we getting timeouts in the delete feed too?

Comment (willkjackson) [reply dataactcore/scripts/pullFPDSData.py]: Yeah, I got them while testing these changes.
